src/server/services/redis_meta_service.h

```diff
@@ -177,6 +177,18 @@ class RedisMetaService : public IMetaService {
   inline void Stop() override;
   ~RedisMetaService() override {}
 
+  void TryAcquireLock(std::string key,
+                      callback_t<bool, std::string> callback_after_try_locked) {
+    // TBD
+    assert(false);
+  }
+
+  void TryReleaseLock(std::string key,
+                      callback_t<bool> callback_after_try_unlocked) {
+    // TBD
+    assert(false);
+  }

```

Comment:
?

Reply:
You could return `Status::NotImplemented` to the callback.

Reply:
Done